src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -20,9 +20,9 @@ class ReportsBox extends React.Component {
   }
 
   async getVisitsByPeriod(domainName, periodSelectedDays) {
```

Comment:
This comment, in fact, is not related to this PR, maybe the previous one.

When a method is called _get..._, in general, it indicates that the method returns something, and it does not have collateral effects.

It is only a coding style issue, but I think that it is an important one, so, it is not necessary to fix it right now, but we should fix it as soon we could.

Reply:
changed it